conference.js

```diff
@@ -2048,8 +2048,9 @@ export default {
         }
 
         APP.settings.setDisplayName(formattedNickname);
-        room.setDisplayName(formattedNickname);
-        APP.UI.changeDisplayName(this.getMyUserId(),
-            formattedNickname);
+        if (room) {
+            room.setDisplayName(formattedNickname);
+            APP.UI.changeDisplayName(this.getMyUserId(), formattedNickname);
```

Comment:
The problem is that this.getMyUserId() will return undefined. When the room is created this code will be anyway executed from UI.initConference.